frontend/src/components/prompt/PromptEntityDelete.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <PopoverPrompt
+    v-model="showDialog"
+    :icon="icon"
+    :title="$tc('components.prompt.promptEntityDelete.title')"
+    :error="error"
+    :submit-action="del"
+    :submit-enabled="submitEnabled && !$slots.error"
+    :submit-label="$tc('global.button.delete')"
+    submit-color="error"
+    :submit-icon="icon"
+    cancel-icon=""
+    :cancel-action="close"
+  >
+    <template #activator="scope">
+      <slot name="activator" v-bind="scope" />
+    </template>
+    <slot>{{ $tc('components.prompt.promptEntityDelete.warningText') }}</slot>
```

Comment:
We could even introduce a "type the name of the entity into this text field before deleting" feature in this component in the future :+1: